src/main/java/com/jfrog/ide/idea/inspections/upgradeversion/UpgradeVersion.java

```diff
@@ -50,11 +52,19 @@ public String getFamilyName() {
 
     @Override
     public void applyFix(@NotNull Project project, @NotNull ProblemDescriptor descriptor) {
-        try {
-            upgradeComponentVersion(project, descriptor);
-        } catch (Exception e) {
-            log.warn("Failed while trying to upgrade component version. Error: " + e);
-        }
+        Task.Backgroundable scanAndUpdateTask = new Task.Backgroundable(project, "Upgrading dependency...") {
+            @Override
+            public void run(@NotNull com.intellij.openapi.progress.ProgressIndicator indicator) {
+                try {
+                    upgradeComponentVersion(project, descriptor);
+                    ApplicationManager.getApplication().invokeLater(() -> descriptor.getPsiElement().getContainingFile().getVirtualFile().refresh(false, false));
+                    log.info("Upgraded " + componentName + " to version " + fixVersion + " successfully.");
+                } catch (Exception e) {
+                    log.error("Failed while trying to upgrade component version. Error: " + e);
```

Comment:
Can we add the component name and version? (If it is not already in the error message itself)